### PR TITLE
Categories

### PR DESCRIPTION
**Pull request by AntoniaBK**
Pull requests should be opened against the `main` branch. For more information on contributing to Lookyloo documentation, see the [Contributor Guidelines](https://www.lookyloo.eu/docs/main/contributor-guide.html).

## Type of change

**Description:**
Fixing the categorization and
adding a checklist to categorize quickly

**Select the type of change(s) made in this pull request:**
- [ ] Bug fix *(non-breaking change which fixes an issue)*
- [x ] New feature *(non-breaking change which adds functionality)*
- [ ] Documentation *(change or fix to documentation)*

---------------------------------------------------------------------------------------------------------

Fixes #issue-number

## Proposed changes 

*
*
*

### CODE REVIEW

File: website/web/templates/tree.html [hunk 1]
```diff
@@ -917,6 +981,7 @@ <h5 class="modal-title" id="screenshotModalLabel">Screenshot</h5>
   </div>
   </div>
 </div>
+############# got it  ####
```
Comment (Rafiot) [inline website/web/templates/tree.html:984]: You probably wait to remove that one?

File: website/web/__init__.py [hunk 2]
```diff
@@ -660,19 +660,31 @@ def historical_lookups(tree_uuid: str) -> str | WerkzeugResponse | Response:
...
 def categories_capture(tree_uuid: str, query: str) -> str | WerkzeugResponse | Response:
     if not enable_categorization:
         return redirect(url_for('tree', tree_uuid=tree_uuid))
-    current_categories = lookyloo.categories_capture(tree_uuid)
     matching_categories = None
-    if query:
+    if 'verification-status' in request.form:
+        # fast categories
+        categories = []
+        possible_ctgs = {'legitime': ["parking-page", "default-page", 'institution', 'captcha', 'authentication-form', 'adult-content', 'shop'],
```
Comment (Rafiot) [inline website/web/__init__.py:672]: Say "legitimate" instead of "legitime" (I think?)

**Review by Rafiot — COMMENTED** [review]
Looks good to me. I haven't tested it yet, but as long as it works for you, we can probably merge it and improve it over time.

File: website/web/__init__.py (hunk 2 above)
Comment (AntoniaBK) [reply website/web/__init__.py:672]: True, I've always liked inventing words

Comment (Rafiot) [reply website/web/__init__.py:672]: That's also the french one ;)

**Pull request merged**